### ISSUE TITLE
Added additional derives for Input structures

### ISSUE THREAD

**Issue by andreevlex** [graphql-rust/graphql-client#195]
Exists struct 

```rust
pub struct UpdateUserInput {
    pub first_name: Option<String>,
    pub last_name: Option<String>,
};
```

I am want write:

```rust
#[derive(GraphQLQuery)]
#[graphql(
    schema_path = "graphql/schema.json",
    query_path = "graphql/queries/update_user.graphql",
    input_derives = "Debug, Default",
    response_derives = "Debug, PartialEq",
)]
pub struct UpdateUserMutation;

let input = UpdateUserInput {
    first_name: "Foo".to_string(),
    ..UpdateUserInput::default()
}
```

**Comment by tomhoule** [comment]
Good point, this is not implemented. We definitely should support that, I'll see if I can work on it soon, but please anyone else feel free to get to this first. The implementation shouldn't be very different from what we have for `response_derives`.

**Comment by davidgraeff** [comment]
Default will only work for simple schemas. As soon as an enum is involved it fails.
Because I'm in the same situation that I don't want to specify all the optional fields in an input type, I have created #260. Does that fix the issue for you?

**Comment by mathstuf** [comment]
Doesn't `additional_derives` handle this already? Also see #249 (where I speculate whether `additional_derives` should be split into `response` and `variables` halves).

**Comment by mathstuf** [comment]
@tomhoule Duplicate of #249.

**Comment by mathstuf** [comment]
Closing as fixed by #285.
